LICENSE.txt

```diff
@@ -0,0 +1,9 @@
+The MIT License (MIT)
+
+Copyright (c) 2016 Yuichi Saito
```

Comment:
Can you change this to Wantedly, Inc? What you've done here is basically belongs to company. Since this license is MIT, this is not big issue though.